src/App.css

```diff
@@ -72,3 +65,54 @@ section {
   opacity: 0.01;
   transition: opacity 300ms ease-in;
 }
+
+/* Waveform */
+#titles, [id*="waveform-visualiser"] {
+  width: 100%;
+}
+
+[id*="waveform-visualiser"] [class*="-container"] {
+  box-shadow: 3px 3px 20px #919191;
+  margin: 0 0 24px 0;
+  -moz-box-shadow: 3px 3px 20px #919191;
+  -webkit-box-shadow: 3px 3px 20px #919191;
+  line-height: 0;
+}
```

Comment:
In general, is it possible to use direct class names in CSS references?   Once you get into Hyrax code, the Hound code quality tool will complain about CSS declarations like this...  just a heads up.

Reply:
It is possible using [attribute*=value] selector in CSS. I guess it is not a good way of doing this. I will change this to a more generic approach using something like an HTML class.

Reply:
Hi @Dananji .So best practice for CSS, and CSS linters (which will complain about certain things), is to use CSS class selectors in almost all cases.  The preferred format is something like `something-wrapper`, or `something-button-group`, `something-button`....to that effect.